workshop/SCHEDULE.md

```diff
@@ -3,38 +3,38 @@ title: Advanced scRNA-seq Workshop Schedule
 nav_title: Schedule
 ---
 
-*Preliminary workshop schedule, subject to change!*
+The planned schedule for the Jan 31-Feb 2, 2023 Data Lab Training Workshop appears below.
 
 | Time      | Topic                                                            |
 |------------------|------------------------------------------------------|
 | **Day 1** | **2023-01-31**                                                   |
 | 9:00 AM   | Welcome, Introductions and Getting Started   <br> [Workshop Introduction slides (PDF)](../slides/2023-01-31_a_Workshop_Intro.pdf)  |
-| 10:00 AM  | Introduction/Refresher to Single-cell Analysis with R and Bioconductor  <br> [Single-cell Analysis with R slides (PDF)](../slides/2023-01-31_b_scRNAseq_intro.pdf)    |
+| 10:00 AM  | Introduction/Refresher to Single-cell Analysis with R and Bioconductor  <br> [Single-cell Analysis with R slides (PDF)](../slides/2023-01-31_b_scRNAseq_intro.pdf)  <br> [Single-cell Analysis with R instruction notebook](http://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq-advanced/01-read_filter_normalize_scRNA.nb.html)  |
```

Comment:
Should this matter on most modern web browsers? I don't think so, they'd probably do the `http` -> `https` for you but just in case...

```suggestion
| 10:00 AM  | Introduction/Refresher to Single-cell Analysis with R and Bioconductor  <br> [Single-cell Analysis with R slides (PDF)](../slides/2023-01-31_b_scRNAseq_intro.pdf)  <br> [Single-cell Analysis with R instruction notebook](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq-advanced/01-read_filter_normalize_scRNA.nb.html)  |
```